src/test/features/first-contact/routes/claim-summary.ts

```diff
@@ -6,20 +6,21 @@ import { attachDefaultHooks } from '../../../routes/hooks'
 import '../../../routes/expectations'
 import { checkAuthorizationGuards } from './checks/authorization-check'
 
-import { ErrorPaths, Paths } from 'first-contact/paths'
+import { ErrorPaths, Paths as DefendantFirstContactPaths } from 'first-contact/paths'
```

Comment:
why make this change? doesn't look like it adds any benefit

Reply:
I was getting errors with TSLint as i had

Reply:
ErrorPaths, Paths then?